Integrations/integration-Threat_Crowd.yml

```diff
@@ -166,6 +182,48 @@ script:
         delete(args.file);
     }
 
+    function calculateDBotScore(res, args, commandData) {
+        var dbotScore = 0;
+        if (res.response_code == 1){
```

Comment:
should it be 1? seems strange, maybe 200?

Reply:
res here is the body of the result that came back from http method.
this api returns 0 in failure and 1 in success   
 
- success:
![image](https://user-images.githubusercontent.com/50294648/63748781-ad601580-c8b2-11e9-93c2-ed7863361095.png)

- failed request or no results :
![image](https://user-images.githubusercontent.com/50294648/63748795-b51fba00-c8b2-11e9-9742-d8ffe9d30b81.png)